src/hotspot/share/prims/jvmti.xml

```diff
@@ -1004,7 +1004,8 @@ jvmtiEnv *jvmti;
     In some cases, <jvmti/> functions allocate memory that your program must
     explicitly deallocate. This is indicated in the individual <jvmti/>
     function descriptions.  Empty lists, arrays, sequences, etc are
-    returned as <code>nullptr</code>.
+    returned as <code>nullptr</code> which is C programming language
+    <code>null</code> pointer.
```

Comment:
Shouldn't this be "NULL"?  In any case, I think it would be helpful to expand this a bit to make it clear that usages of "nullptr" in parameter and error descriptions should be read or treated as  "NULL" when developing an agent in C rather than C++.

Reply:
Yes, I think it should by NULL.

Reply:
Thank you for suggestions. I've changed it as Kim suggested below.
Please, let me know if it is not good enough, or some additions are needed.